### PR TITLE
Fixed issue with exception in beforeEach hook #24

### PR DESCRIPTION
**Pull request by stalniy**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -26,6 +26,10 @@ function isInvalidSuite(suite) {
   return suite.title === '' || suite.tests.length === 0 && suite.suites.length === 0;
 }
 
+function lastSuite(suites) {
```
Comment (clayreimann) [inline index.js]: This function doesn't appear to be used.

Comment (stalniy) [inline index.js]: @clayreimann fixed

**Comment by clayreimann** [comment]
Needs rebase now, but looks good

**Comment by stalniy** [comment]
@clayreimann rebased

**Comment by clayreimann** [comment]
:+1: Nice!

**Pull request merged**